__tests__/app.test.js

```diff
@@ -45,25 +37,108 @@ describe('/api', () => {
     });
 });
 
-describe('/api/articles/:article_id', () => {
+describe('GET /api/articles/:article_id', () => {
     test('GET: 200 an article object with all associated properties', () => {
```

Comment:
We need to make sure to add tests here for a 400 and 404 error
Status 400, invalid ID, e.g. string of "not-an-id"
Status 404, non existent ID, e.g. 0 or 9999

Reply:
Actually I can see the above are within your error handling block, however I personally would be included to put the ones to do with invalid ids, under this describe block